komodoenv/update.py

```diff
@@ -10,22 +10,31 @@
 import re
 import sys
 import shutil
+import platform
 from typing import List
 from argparse import ArgumentParser
 from textwrap import dedent
 
-
 try:
     from distro import id as distro_id, version_parts as distro_versions
 except ImportError:
-    # The 'distro' package isn't installed. Pretend we're on RHEL7.
+    # The 'distro' package isn't installed.
     #
-    # yum install python36-distro
     def distro_id():
         return "rhel"
 
-    def distro_versions():
-        return ("7", "0", "0")
+    if "el7" in platform.platform():
```

Comment:
This could technically be compacted some, very unsure if this makes any sense though;

```
if platform.platform() in ["el7", "el8"]:
     def distro_versions():
        return (platform.platform()[2], "0", "0")
```